src/odemis/driver/test/smaract_test.py

```diff
@@ -332,6 +333,21 @@ def test_pivot_set(self):
             self.dev.updateMetadata({model.MD_PIVOT_POS: old_pivot})
             self.dev.moveRelSync({"x": 0})
 
+    def test_reference_and_deactivate_move(self):
+        # Set a deactive position and check to be sure that the controller moves to this location
+        # after a reference move
+        # TODO: UIse real physical deactiva position
```

Comment:
It'd be difficult to find a position which works on any system any way. Let's drop this TODO.